lisp/ui/layoutselect.py

```diff
@@ -99,8 +100,12 @@ def show_description(self):
         )
 
     def open_file(self):
+        last_session_path = AppConfig().get("session.last_path", os.getenv("HOME"))
```

Comment:
Here it could be convenient to keep a reference of `AppConfig` instead of calling it multiple time (e.g. `conf = AppConfig()`).
This change affect also the lines 109/110.